src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2484,7 +2491,96 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/calculateSize':
+    post:
+      tags:
+        - nodes
+      summary: Calculate a folder size
+      description: |
+        
+        This may be executed against a node specified by **nodeId** and this nodeId must be a valid node ID.
+        Furthermore, this POST endpoint will be executed **asynchronously** with a `202` HTTP response signifying that
+        the request has been accepted successfully.
```

Comment:
Again, it will return the **nodeId** for which the Action was initiated and start the execution asynchronously.
Similarly, we have a **GET endpoint** to check the status and output for the specified Node.

Reply:
I think you should add some documentation for this case. If the POST is called twice then I would find it surprising that the same id is returned both times, and I'm not clear which result I will get when calling the GET (e.g. Is the existing job cancelled? Does the new job overwrite the output of the first job?)

Reply:
Previously, we used the nodeService property attribute to store the result output.
But now we'll look at the implementation with an **execution job ID** instead of a node ID and return the **execution Job ID** as a response.

Reply:
If the POST is called twice then I would find it surprising that the same id is returned both times, and I'm not clear which result I will get when calling the GET (e.g. Is the existing job cancelled? Does the new job overwrite the output of the first job?)
 [Updated Remarks]: Now, we are considering executionId instead of nodeId. Thus, existing job will not be cancelled if we triggered POST twice [ as its working with different execution Id ].
 And the same will not overwrite the output of the first job.